pkg/controller/pv_control.go

```diff
@@ -255,4 +295,39 @@ func (c *FakePVControl) UpdateMetaInfo(obj runtime.Object, pv *corev1.Persistent
 	return pv, c.PVIndexer.Update(pv)
 }
 
+func (c *FakePVControl) PatchPVClaimRef(obj runtime.Object, pv *corev1.PersistentVolume, pvcName string) error {
+	defer c.updatePVTracker.Inc()
+	if c.updatePVTracker.ErrorReady() {
+		defer c.updatePVTracker.Reset()
+		return c.updatePVTracker.GetError()
+	}
+	pv.Spec.ClaimRef.Name = pvcName
+
+	return c.PVIndexer.Update(pv)
+}
+
+// CreatePV create new pv
+func (c *FakePVControl) CreatePV(_ runtime.Object, pv *corev1.PersistentVolume) error {
+	defer c.createPVTracker.Inc()
+	if c.createPVTracker.ErrorReady() {
+		defer c.createPVTracker.Reset()
+		return c.createPVTracker.GetError()
+	}
+
+	return c.PVIndexer.Add(pv)
+}
+
+func (c *FakePVControl) GetPV(name string) (*corev1.PersistentVolume, error) {
+	defer c.updatePVTracker.Inc()
```

Comment:
Why call `updatePVTracker` here?

Reply:
I refer the pvc code  https://github.com/pingcap/tidb-operator/blob/2ef71285d46e6b8ad962343d4b3c2ab8d4afa133/pkg/controller/pvc_control.go#L272.